bundles/framework/personaldata/resources/scss/personaldata.scss

```diff
@@ -88,4 +88,9 @@ textarea#view_description {
 /* Temporary hack for too-long tabHeader */
 .personaldata .tabsHeader {
     margin-right: 21px;
-}
\ No newline at end of file
+}
+
+.table-actions {
+    display: flex;
+    justify-content: space-evenly;
+}
```

Comment:
Let's do this with styled-components instead.

Reply:
Added export under oskari-ui/Table.